cmake/PostgreSQLExtension.cmake

```diff
@@ -255,6 +255,16 @@ function(add_postgresql_extension NAME)
                 PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
     endif()
 
+    # Check that the extension has no conflicting symbols with the Postgres binary
+    # otherwise the linker might use the symbols from Postgres
+    find_package(Python COMPONENTS Interpreter REQUIRED)
+    execute_process(COMMAND dirname ${PG_CTL} OUTPUT_VARIABLE POSTGRES_BINARY)
+    get_filename_component(POSTGRES_BINARY ${PG_CTL}/../postgres ABSOLUTE)
+    set(SO_FILE "${CMAKE_CURRENT_BINARY_DIR}/${_ext_TARGET}${_suffix}")
+    add_custom_target(omni_check_symbol_conflict_${_ext_TARGET} ALL
```

Comment:
ALL is probably not the target to create the dependency on... need a bit help with this.

Reply:
Target file can be obtained more easily (I think) with `$<TARGET_FILE:target>`

As for the dependency, maybe `(package_${_ext_TARGET}_extension`? It's not going to get during the build process, though but packaging. 🤔